utils_cv/tracking/model.py

```diff
@@ -177,15 +177,45 @@ def fit(
         trainer.set_device(
             self.opt.gpus, self.opt.chunk_sizes, self.opt.device
         )
-
+        
+        # initialize loss vars
+        self.losses_dict = {key: [] for key in ['loss', 'hm_loss', 'wh_loss', 'off_loss', 'id_loss']} 
+        self.epochs = []
+        
         # training loop
         for epoch in range(self.start_epoch + 1, self.opt.num_epochs + 1):
+            print("Training in epoch ", epoch,"...")
             mark = epoch if self.opt.save_all else "last"
             log_dict_train, _ = trainer.train(epoch, train_loader)
             if epoch in self.opt.lr_step:
                 lr = self.opt.lr * (0.1 ** (self.opt.lr_step.index(epoch) + 1))
                 for param_group in optimizer.param_groups:
                     param_group["lr"] = lr
+                    
+            # store losses in each epoch            
+            self.epochs.append(epoch)
+            for k, v in log_dict_train.items():
+                if k in ['loss', 'hm_loss', 'wh_loss', 'off_loss', 'id_loss']:
+                    self.losses_dict[k].append(v)
+
+    def plot_training_losses(self, figsize: Tuple[int, int] = (10, 5))->None: 
```

Comment:
Please for function comments use the same syntax as everywhere in the code, as e.g. the fit method, ie:
        """
        <text description>
       
        Args:
            figsize: <text>
        """